examples/35_gemm_softmax/gemm_softmax.cu

```diff
@@ -159,6 +165,8 @@ struct Options {
   /// Returns true if the environment and Toolkit support this
   bool supported(bool verbose = true) const {
 
+#if !defined(CUTLASS_ENABLE_SYCL)
```

Comment:
This is only temporary needed until we rework how `__CUDACC_VER_MAJOR__` works for syclcompat, right? If so a code comment would be good so that we can find those easily in the future.